### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by prryplatypus**

### CODE REVIEW

**Review by ChihweiLHBird — APPROVED** [review]
lgtm

File: README.rst [hunk 1]
```diff
@@ -163,8 +165,3 @@ Contribution
 ------------
 
 We are always happy to have new contributions. We have `marked issues good for anyone looking to get started <https://github.com/sanic-org/sanic/issues?q=is%3Aopen+is%3Aissue+label%3Abeginner>`_, and welcome `questions on the forums <https://community.sanicframework.org/>`_. Please take a look at our `Contribution guidelines <https://github.com/sanic-org/sanic/blob/master/CONTRIBUTING.rst>`_.
-
-.. |Linode| image:: https://www.linode.com/wp-content/uploads/2021/01/Linode-Logo-Black.svg
-    :alt: Linode
-    :target: https://www.linode.com
-    :width: 200px
```
Comment (ahopkins) [inline README.rst:170]: Please do not remove this.

**Review by ahopkins — CHANGES_REQUESTED** [review]
Add back Linode logo. When current deal is done we will deal with it.

File: README.rst (hunk 1 above)
Comment (prryplatypus) [reply README.rst:170]: I didn't remove it. I just moved it up with all the others :D

Comment (prryplatypus) [reply README.rst:170]: imagen

**Review by ahopkins — APPROVED** [review]

**Pull request merged**